slackbridge/bots.py

```diff
@@ -93,9 +96,19 @@ def _change_presence(self, user_bot):
             elif self.raw_message['presence'] == 'active':
                 user_bot.back()
 
+        def _irc_action(self, channel_name, user_bot):
```

Comment:
The method title seems to give an impression that it's a base method to extend off of. Maybe make it a bit more explicit with "_irc_me_action" or something along those lines?

Reply:
Good point, that would be clearer, I'll change it.